aries/aries_vcx_core/src/wallet/base_wallet/record.rs

```diff
@@ -28,3 +31,54 @@ impl Record {
         &self.tags
     }
 }
+
+#[derive(Debug, Default, Clone, TypedBuilder)]
+pub struct PartialRecord {
+    category: Option<String>,
+    name: String,
+    value: Option<String>,
+    #[builder(default)]
+    tags: Option<RecordTags>,
+}
+
+impl PartialRecord {
+    pub fn value(&self) -> &Option<String> {
+        &self.value
+    }
+
+    pub fn name(&self) -> &str {
+        &self.name
+    }
+
+    pub fn category(&self) -> &Option<String> {
+        &self.category
+    }
+
+    pub fn tags(&self) -> &Option<RecordTags> {
+        &self.tags
+    }
+
+    #[cfg(feature = "vdrtools_wallet")]
```

Comment:
Should be implemented in the `indy` module.

Reply:
Moved.